src/GitHub.StartPage/StartPagePackage.cs

```diff
@@ -144,6 +146,8 @@ public class GitHubContainerProvider : ICodeContainerProvider
                         result.Repository.Name,
                         result.BasePath,
                         progress);
+
+                    usageTracker.IncrementCounter(x => x.NumberOfStartPageClones).Forget();
```

Comment:
I'm using `Forget()` from `Microsoft.VisualStudio.Threading` to avoid creating a dependency on `GitHub.Extensions`.